sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewView.groovy

```diff
@@ -81,6 +86,7 @@ class ProjectOverviewView extends VerticalLayout implements HasHotbar, HasTitle
 
     private void initLayout(){
         setupProjects()
+        setupSubscriptionCheckBox()
```

Comment:
In case the checkboxes are created/hashed by the factory. you do not need this step as the grid will automatically call this functionality whenever a checkbox is to be shown.